api/models.py

```diff
@@ -147,3 +148,22 @@ class User(AbstractUser):
     USERNAME_FIELD = 'email'
     REQUIRED_FIELDS = ['cohort', 'slack_handle']
     objects = UserManager()
+
```

Comment:
while we're here.. can we add `verbose_name_plural = 'All Users'` for the User class? seems like it's also listing security users under `Users`

Reply:
Sure. I will add this.

Reply:
Done.